Singleton/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include_directories(inc)
+
+set(Singleton src/Singleton.cpp inc/Singleton.hpp)
```

Comment:
The .hpp file is not necessary to be grouped with the source files, since you have included the directory.

Reply:
Ok, thank you.